packages/scaffold-ui/test/partials/w3m-default-account-widget.test.ts

```diff
@@ -255,10 +255,6 @@ describe('W3mAccountDefaultWidget', () => {
       await disconnectButton?.click()
 
       expect(ConnectionController.disconnect).toHaveBeenCalled()
-      expect(EventsController.sendEvent).toHaveBeenCalledWith({
-        type: 'track',
-        event: 'DISCONNECT_SUCCESS'
-      })
```

Comment:
is this ok?

Reply:
Yes, we shouldn't call disconnect success in UI, since we have single point to disconnect users which is `ChainController.disconnect`, and it's enough to call this there